timApp/static/scripts/tim/item/manageCtrl.ts

```diff
@@ -764,6 +764,14 @@ export class PermCtrl implements IController {
     loggedIn() {
         return Users.isLoggedIn();
     }
+
+    /**
+     * Returns the path to a document specified by doc_id
+     * @param doc_id
+     */
+    getTranslationPath(doc_id: number) {
+        return this.translations.find((tr) => tr.id == doc_id)?.path ?? "";
```

Comment:
Voisiko tuohon `""`:n muttaa vaikkapa `this.item.path`? Silloin jos jostain syystä ei käännöstä löydy, niin linkiksi tulee ainakin nykyinen dokumentti. Ainakin se saattaa olla järkevämpi vaihtoehto kuin se, että se vie.

Eli

```suggestion
        return this.translations.find((tr) => tr.id == doc_id)?.path ?? this.item.path;
```